.github/workflows/minor_version_release.yml

```diff
@@ -25,17 +21,52 @@ jobs:
           python -m pip install --upgrade pip
           pip install pydoc-markdown==4.5.1
 
-      - name: Convert version name to regular semver (e.g. v1.10.x -> v1.10.0)
-        id: regular_semver
-        env:
-          version: ${{github.event.inputs.version}}
+      - name: Define all versions
+        id: versions
+        shell: bash
+        # We only need `major.minor` in Readme so we cut the full version string to the first two tokens
         run: |
-            echo $version | sed -e 's/x$//' | { var=$(cat); echo "VERSION=v${var}0" >> $GITHUB_ENV; }
-            echo $version | sed -e 's/x$//'
+          git fetch
+          git checkout main
+          echo "::set-output name=new_release_version::$(cat VERSION.txt | cut -d "." -f 1,2)"
```

Comment:
Let's call this output `current_release_minor` to avoid confusion:
```
echo "::set-output name= current_release_minor::$(cat VERSION.txt | cut -d "." -f 1,2)"
```

Reply:
Ok, I have changed the variable name